File: Problemas/281A - Word Capitalization/281A_Word_Capitalization.cpp
// 281A - Word Capitalization
// https://codeforces.com/problemset/problem/281/A

#include <stdio.h>
#include <stdlib.h>
#include <bits/stdc++.h>
#include <cmath>
using namespace std;

int main(){

    // Input:
    // 1) palabra = Palabra a capitalizar
    string palabra;
    cin >> palabra;

    // Capitaliza la 1° letra de la palabra
    palabra[0] = toupper(palabra[0]);

    // Imprime la palabra capitalizada
    cout << palabra << endl;
    
    return 0;
}

File: Problemas/281A - Word Capitalization/281A_Word_Capitalization.py
# 281A - Word Capitalization
# https://codeforces.com/problemset/problem/281/A

# Input:
# 1) palabra - Palabra a capitalizar
palabra = input()

# Convierte la primera letra a mayúsculas
primera_letra = palabra[0].upper()

# Imprime la palabra capitalizada
# palabra[1:] = Resto que letras que no fueron capitalizadas
print(primera_letra + palabra[1:])
